app/package.json

```diff
@@ -6,12 +6,12 @@
   "license": "None",
   "private": true,
   "dependencies": {
-    "@arizeai/components": "^1.2.2",
+    "@arizeai/components": "^1.3.3",
     "@arizeai/openinference-semantic-conventions": "^0.1.0",
     "@arizeai/point-cloud": "^3.0.4",
-    "@codemirror/autocomplete": "^6.12.0",
-    "@codemirror/lang-json": "^6.0.1",
-    "@codemirror/lang-python": "^6.1.3",
+    "@codemirror/autocomplete": "6.12.0",
+    "@codemirror/lang-json": "6.0.1",
+    "@codemirror/lang-python": "6.1.3",
```

Comment:
Did these need to be pinned exactly in this way?

Reply:
Yeah for now